src/winrpath.cxx

```diff
@@ -95,8 +98,9 @@ int get_padding_length(const std::string &name)
     int c = 0;
     std::string::const_iterator p = name.cbegin();
     p+=2;
-    while(p != name.end() && *p == '/') {
+    while(p != name.end() && *p == '\\') {
         ++c;
+        ++p;
```

Comment:
I assume a padded path might look like e.g. `C:\\foo\bar\\\\\\\\\\`. This seems like it would stop early.

Reply:
Padded paths are only ever of the form: `C:\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\foo\bar` when generated by this wrapper.

Reply:
OK that is a useful note

* That would be great as a comment on `pad_path`
* In particular that generally makes sense for `<drive-letter>:\`-style paths, but it seems awkward for network drives; or am I thinking about it wrong? Maybe an example for each type of path you intend to support would also be useful in the docstring

Reply:
Added the comment. This would handle network drives poorly, that said, I actually don't know the implications of padding network drives, so I'll need to experiment.